src/main/java/com/redhat/quarkus/mandrel/collector/report/model/ImageStats.java

```diff
@@ -70,6 +70,9 @@ public class ImageStats extends TimestampedEntity {
     @JsonProperty("generator_version")
     private String graalVersion;
 
+    @JsonProperty("test_version")
+    private String testVersion;
+
```

Comment:
I don't think it makes sense to keep it here. It should be in `auxiliaryInfo` now.